app/policies/pair_request/acceptance_policy.rb

```diff
@@ -0,0 +1,19 @@
+class PairRequest::AcceptancePolicy < ApplicationPolicy
+  attr_reader :user, :pair_request
+
+  def initialize(user, pair_request)
```

Comment:
If you look in `ApplicationPolicy`, the initializer and `attr_reader` is already set up, and since this class inherits from `ApplicationPolicy`, this stuff doesn't need to be repeated here.

If you look in `PairRequestPolicy` you'll see a nice pattern to use. We just alias the `record` method to be the name of the object we're working with so it's easier to understand exactly what object is involved.